src/pynwb/tests/test_example_usage.py

```diff
@@ -244,3 +246,4 @@ def test_example_usage():
         assert read_optogenetic_epochs_table[0, "number_trains"] == 1
         assert read_optogenetic_epochs_table[0, "intertrain_interval_in_ms"] == 0.0
         assert read_optogenetic_epochs_table[0, "power_in_mW"] == 77.0
+        assert read_optogenetic_epochs_table.optical_fiber_locations_table_region.data[:] == [0]
```

Comment:
```suggestion
        assert read_optogenetic_epochs_table.optical_fiber_locations_table_region.data[:] == [0]
        assert read_optogenetic_epochs_table.optical_fiber_locations_table_region.table is optical_fiber_locations_table
```

A little redundant but I like to add this check just to make sure the linking worked correctly on the HDMF side.